core/CPUTopology.cpp

```diff
@@ -358,6 +346,37 @@ void olympia::CoreTopologySimple::bindTree(sparta::RootTreeNode* root_node)
                 bind_ports(exe_flush_out, flush_manager_in);
             }
         }
+        auto issue_queue_topology =
+            olympia::coreutils::getPipeTopology(root_node->getChild(core_node), "issue_queue_topology");
+        for(size_t i = 0; i < issue_queue_topology.size(); ++i){
+            std::string unit_name = "iq" + std::to_string(i);
```

Comment:
Can be `const`

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)